failure in cm/uct_p2p_am_test.am_bcopy/1

http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1641/label=master,worker=2/consoleFull
```
01:47:42 [ RUN      ] cm/uct_p2p_am_test.am_bcopy/1
01:47:42 [     INFO ] 0 214 1000x{0..214} /var/lib/jenkins/jobs/gh-ucx-pr/workspace/label/master/worker/2/contrib/../test/gtest/uct/test_p2p_am.cc:184: Failure
01:47:44 Value of: m_am_count
01:47:44   Actual: 2
01:47:44 Expected: 1u
01:47:44 Which is: 1
01:47:47 nocomp 14 
01:47:47 [  FAILED  ] cm/uct_p2p_am_test.am_bcopy/1, where GetParam() = cm/mlx5_0:1/loopback (4989 ms)
...
02:01:01  1 FAILED TEST
02:01:01 make: *** [test] Error 1
```

--- a/test/gtest/uct/test_p2p_am.cc
+++ b/test/gtest/uct/test_p2p_am.cc
@@ -181,7 +181,7 @@ public:
             short_progress_loop();
         }
 
-        if (receiver().iface_attr().cap.flags & UCT_IFACE_FLAG_AM_DUP) {
+        if (!(receiver().iface_attr().cap.flags & UCT_IFACE_FLAG_AM_DUP)) {
             sender().flush();
             EXPECT_EQ(1u, m_am_count);
         } else {

